### PR TITLE
Add asset id OneAtlas up5106

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
Pull Request
============

> Tracked in JIRA by [IN-5106](https://geoinformationstore.atlassian.net/browse/IN-5106)

### Scope of the PR

Remove mentions of order_ids. Add asset_id.

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly
- [ ] Update gists FYI @perusio

### CODE REVIEW

**Review by markusuwe — APPROVED** [review]
LGTM

File: source/going-further/download-blocks-tutorial-spot-pleiades.rst [hunk 1]
```diff
@@ -196,39 +196,17 @@ Download the image
 Now that you have the price estimate we can proceed and acquire the
 image. To do this we rerun the job as a real job by clicking on the
 **Run as real job** button on the console job details page. When the
-job is launched, the upstream creates an order ID. This is the unique
-identifier for the downloaded image.
+job is launched, the upstream creates an asset ID in your storage.
```
Comment (perusio) [inline source/going-further/download-blocks-tutorial-spot-pleiades.rst:199]: job is launched and an asset ID is created that uniquely identifies this asset in the collection of assets available in your User Storage.

**Review by perusio — CHANGES_REQUESTED** [review]
Minor things: please see below.

File: source/going-further/download-blocks-tutorial-spot-pleiades.rst (hunk 1 above)
Comment (rodrigoalmeida94) [reply source/going-further/download-blocks-tutorial-spot-pleiades.rst:199]: Added

**Review by thaisbendixen — APPROVED** [review]
LGTM

**Review by perusio — APPROVED** [review]
LGTM.

**Pull request merged**